apps/designer/app/designer/features/style-panel/shared/css-value-input/css-value-input.tsx

```diff
@@ -200,8 +200,8 @@ type CssValueInputProps = {
 
 export const CssValueInput = ({
   property,
-  value = defaultValue,
-  keywords = defaultKeywords,
+  value = unsetValue,
+  keywords = [],
```

Comment:
The previous version when array was created outside was a bit better because now a new value will be created on every render, which may cause extra updates down the line if `keywords` are used as a dependency somewhere.

Not sure whether this is a concern in this case, but I'm guessing that's why it was outside before

Reply:
nah, don't think its a concern, was outside because it was used inside as a fallback and would actually prevent an update in case keywords were default, so that its the same ref, but its not used any more twice, so I moved it